odin-dynamic/src/main/scala/com/permutive/logging/dynamic/odin/DynamicOdinLogger.scala

```diff
@@ -71,12 +71,27 @@ object DynamicOdinConsoleLogger {
 
   case class RuntimeConfig(
       minLevel: Level,
-      levelMapping: Map[String, Level] = Map.empty
+      levelMapping: Map[String, LevelConfig] = Map.empty
   )
   object RuntimeConfig {
     implicit val eq: Eq[RuntimeConfig] = cats.derived.semiauto.eq
   }
 
+  sealed trait LevelConfig {
+    def toLevel: Option[Level]
+  }
+
+  object LevelConfig {
+    case object Trace extends LevelConfig { val toLevel = Some(Level.Trace) }
+    case object Debug extends LevelConfig { val toLevel = Some(Level.Debug) }
+    case object Info extends LevelConfig { val toLevel = Some(Level.Info) }
+    case object Warn extends LevelConfig { val toLevel = Some(Level.Warn) }
+    case object Error extends LevelConfig { val toLevel = Some(Level.Error) }
+    case object Off extends LevelConfig { val toLevel = None }
+
+    implicit val eq: Eq[LevelConfig] = cats.derived.semiauto.eq
+  }
```

Comment:
Having an ADT here is nice, but it makes it hard to evolve the config model in a bincompat fashion, I think it might be better to use a value class here with a smart constructor that checks the string value.